pyscriptjs/src/components/pyscript.ts

```diff
@@ -113,9 +162,12 @@ function createElementsWithEventListeners(interpreter: InterpreterClient, el: El
         ensureUniqueId(el as HTMLElement);
     }
     elementInterpreter.set(el, interpreter);
-    // Note: this is *NOT* a misused-promise, this is how async events work.
+
+    // Note: these are *NOT* misused-promises, this is how async events work.
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
+    if (type.slice(-5) === '-code') el.addEventListener(type.slice(0, -5), pyScriptCodeRunner);
```

Comment:
two slices seem unnecessary ... how about `type.endsWith('-code')` here and we slice only if needed?

Reply:
Love it.